src/vs/workbench/contrib/interactive/browser/interactive.contribution.ts

```diff
@@ -791,11 +791,6 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).regis
 			type: 'boolean',
 			default: true,
 			markdownDescription: localize('interactiveWindow.alwaysScrollOnNewCell', "Automatically scroll the interactive window to show the output of the last statement executed. If this value is false, the window will only scroll if the last cell was already the one scrolled to.")
-		},
-		[InteractiveWindowSetting.interactiveWindowRestore]: {
```

Comment:
I see you had a deprecation message in here before, why not show that for at least one release? The setting previously existed for 11 months

Reply:
from @rebornix's input, we can just remove it since it won't do anything and just mention it in the release notes.